06_deploy_bootstrap_vm.sh

```diff
@@ -36,7 +36,7 @@ echo "address=/api.${CLUSTER_DOMAIN}/${API_VIP}" | sudo tee /etc/NetworkManager/
 sudo systemctl reload NetworkManager
 
 # Wait for ssh to start
-while ! $SSH core@$IP id ; do sleep 5 ; done
+$SSH -o ConnectionAttempts=100 core@$IP id
```

Comment:
I looked at one of the CI jobs and this took 27 attempts with 5 second sleep, thats 135 seconds, I guess we should bump that to ConnectionAttempts==200 ?

Reply:
Bumped it up to 500 just to be sure